satpy/utils.py

```diff
@@ -627,6 +628,8 @@ def _sort_files_to_local_remote_and_fsfiles(filenames):
     for f in filenames:
         if isinstance(f, FSFile):
             fs_files.append(f)
+        elif isinstance(f, pathlib.Path):
+            local_files.append(f)
```

Comment:
codecov is complaining...

Reply:
That's weird, because without this addition the tests weren't passing locally.

Reply:
And adding a print inside that branch shows the filename.

Reply:
Lets see if the complaint is still there after the tests complete for the latest changes.

Reply:
Added a separate test in 596aa5966